configs/sst_cs_apps-unwanted-python3-eln.yaml

```
@@ -27,6 +29,7 @@ data:
   unwanted_source_packages:
   - pytz
   - python-chardet
+  - python-py
   - python-toml
   - python-tomli
```

Perhaps keep the same order here?

```suggestion
  - python-toml
  - python-tomli
  - python-py
```